visage_windowing/macos/windowing_macos.mm

```diff
@@ -174,9 +174,22 @@ void setCursorScreenPosition(Point screen_position) {
   }
 
   void setCursorPosition(Point window_position) {
-    CGRect window_bounds = activeWindowBounds();
```

Comment:
This is the only place where activeWindowBounds() is used so could delete that function.